app/views/candidate_interface/guidance/index.html.erb

```diff
@@ -8,47 +8,99 @@
-        when the academic year in England starts. Part time courses are usually 2 years.</p>
-
-      <%= govuk_table do |table|
-            table.with_head do |head|
-              head.with_row do |row|
-                row.with_cell(text: 'Date and time')
-                row.with_cell(text: 'What happens')
-              end
-            end
-
-            table.with_body do |body|
-              body.with_row do |row|
-                row.with_cell(text: @current_timetable.find_opens_at.to_fs(:govuk_date_and_time))
-                row.with_cell do
-                  "Start #{govuk_link_to('finding postgraduate teacher training courses', find_url)} you might want to apply to.".html_safe
-                end
-              end
-              body.with_row do |row|
-                row.with_cell(text: @current_timetable.apply_opens_at.to_fs(:govuk_date_and_time))
-                row.with_cell do
-                  "<p class=\"govuk-body\">Start applying to courses. You can apply to #{ApplicationForm::MAXIMUM_NUMBER_OF_COURSE_CHOICES} courses, they can fill up quickly so you should apply as soon as you can.</p>
-                   <p class=\"govuk-body\">You can withdraw an application at any time and apply to a different course if you want to.</p>
-                   <p class=\"govuk-body\">If training providers do not respond to an application within 30 working days, you will be able to apply to another course while you wait for a response. We will tell you if the provider has not responded in time.</p>
-                   <p class=\"govuk-body\">You will get an email if you receive an offer from a training provider.</p>".html_safe
-                end
-              end
-              body.with_row do |row|
-                row.with_cell(text: @current_timetable.apply_deadline_at.to_fs(:govuk_date_and_time))
-                row.with_cell(text: 'The last day to submit any applications.')
-              end
-              body.with_row do |row|
-                row.with_cell(text: @current_timetable.reject_by_default_at.to_fs(:govuk_date_and_time))
-                row.with_cell(text: "The last day for training providers to make a decision on all applications for courses starting in September #{@recruitment_cycle_year}.")
-              end
-            end
-          end %>
+    <h1 class="govuk-heading-l">
+      <%= t('.heading', this_cycle: @recruitment_cycle_year) %>
+    </h1>
+
+    <p class="govuk-body">
+      <%= t('.you_can_start_applying', this_cycle: @recruitment_cycle_year, last_cycle: @recruitment_cycle_year - 1) %>
+    </p>
+
+    <p class="govuk-body govuk-!-margin-bottom-6">
+      <%= t('.duration') %>
+    </p>
+
+    <div class="app-timeline">
+      <div class="app-timeline__item">
+        <div class="app-timeline__header">
+          <h3 class="app-timeline__title">
+            <%= @current_timetable.find_opens_at.to_fs(:govuk_date_and_time) %>
+          </h3>
```

Comment:
Line 11 is an h1. We can't jump from an h1 to an h3. There has to be an h2 in between.